src/modules/tagInfo.ts

```diff
@@ -67,3 +67,9 @@ export const getTagInfo = async ({
     markup,
   };
 };
+
+export const removeSrcAttribute = (attributes: Record<string, any>) => {
```

Comment:
should this be a generic util?

```
export const filterAttributes(attributes: Record<string,any>, filterFn: (a:string) => boolean) {
  return Object.fromEntries(Object.entries(([a])=>filterFn(a)))
}
```

usage:
```js
const cleanAttributes = filterAttributes(attributes, a => a !=='src');
// or
const attributesToRemove = ['foo','bar];
const cleanMultipleAttributes = filterAttributes(attributes, a => !attributesToRemove.includes(a))
```

that can be reused for global and potentially others?